class7-week3/actions/markAsDone.js

```diff
@@ -0,0 +1,23 @@
+const Todo = require('../models/todo')
+const deserializeTodo = require('../util/deserializeTodo')
+
+module.exports = function markAsDone(request, response) {
+
+  const id = request.params.id
+  const todo = deserializeTodo(request, response)
+  if (todo == null) { return }
+
+  Todo.markAsDone(id, todo.done, (error, todo) => {
```

Comment:
Once you've commented out lines 7 and 8, you'll need to look at this function signature. It's not correct.

Hint: Your functions in todo.js have the *correct* signature - so you should match them (again in markAsNotDone as well).